layouts/partials/navbar.html

```diff
@@ -1,25 +1,28 @@
 <header class="of-header-main">
-    <a href="/" class="of-brand">
-      <picture class="of-brand__picture">
-        <source srcset="/images/logo.svg" media="(min-width: 992px)">
-        <img src="/images/logo-sm.svg" alt="">
-      </picture>
-    </a>
-    <nav class="of-nav-main nav-collapse">
-      <ul class="of-nav-main__items menu-items">
-        <li class="of-nav-main__item"><a class="of-link-list__a {{ if eq .URL "/" }} of-m-active {{end}}" href="/">Home</a></li>
-        {{ $currentPage := . }}
-        {{ range .Site.Menus.main }}
-        <li class="of-nav-main__item"><a class="of-link-list__a{{if or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }} of-m-active{{end}}" href="{{ .URL }}" title="{{ .Title }}">{{ .Name }}</a></li>
-        {{ end }}
-        {{ if  .Site.Params.versions }}
-        <li class="of-nav-main__item">
-        {{ partial "navbar-version-selector.html" . }}
-        </li>
-        {{ end }}
-      </ul>
-    </nav>
-    <div class="of-header-main__search">
-		{{ partial "search-input.html" . }}
-    </div>
-  </header>
\ No newline at end of file
+  <a href="/" class="of-brand">
+    <picture class="of-brand__picture">
+      <source srcset="/images/logo.svg" media="(min-width: 992px)">
+      <img src="/images/logo-sm.svg" alt="">
+    </picture>
+  </a>
+  <nav class="of-nav-main nav-collapse">
+    <ul class="of-nav-main__items menu-items">
+      <li class="of-nav-main__item"><a class="of-link-list__a {{ if eq .URL "/" }} of-m-active {{end}}" href="/">Home</a></li>
+      {{ $currentPage := . }}
+      {{ range .Site.Menus.main }}
+      <li class="of-nav-main__item"><a class="of-link-list__a{{if or ($currentPage.IsMenuCurrent "main" .) ($currentPage.HasMenuCurrent "main" .) }} of-m-active{{end}}" href="{{ .URL }}" title="{{ .Title }}">{{ .Name }}</a></li>
+      {{ end }}
+      {{ if  .Site.Params.versions }}
+      <li class="of-nav-main__item">
+      {{ partial "navbar-version-selector.html" . }}
+      </li>
+      {{ end }}
+    </ul>
+  </nav>
+  <div id="search-box">
+    <!-- SearchBox widget will appear here -->
```

Comment:
try dropping this in here
`{{ partial "search-input.html" . }}`

Reply:
@matthewcarleton do you mean like thishttps://github.com/operator-framework/olm-docs/blob/d352a7f7771ae32b6fa1308143e85e2cc0e72335/layouts/partials/navbar.html#L24? 

Isn't that just calling `search-input.html`? 
I was under the impression that this algolia search box will be replacing the `search-input.html`